include/mbedtls/error.h

```diff
@@ -105,6 +105,7 @@
  * CIPHER    6   8 (Started from 0x6080)
  * SSL       6   24 (Started from top, plus 0x6000)
  * SSL       7   32
+ * PKCS7     8   10
```

Comment:
Unfortunately, this is a 3-bit value, so 8 doesn't fit. PKCS7 will have to squeeze in with some other existing high-level module number.

Hopefully in Mbed TLS 3 we'll get rid of this error code structure.